bazel/external/quiche.BUILD

```diff
@@ -253,6 +254,48 @@ cc_library(
     deps = [":quic_platform_export"],
 )
 
+cc_library(
+    name = "epoll_server_platform",
+    testonly = 1,
+    hdrs = [
+        "quiche/epoll_server/platform/api/epoll_address_test_utils.h",
+        "quiche/epoll_server/platform/api/epoll_bug.h",
+        "quiche/epoll_server/platform/api/epoll_expect_bug.h",
+        "quiche/epoll_server/platform/api/epoll_export.h",
+        "quiche/epoll_server/platform/api/epoll_logging.h",
+        "quiche/epoll_server/platform/api/epoll_ptr_util.h",
+        "quiche/epoll_server/platform/api/epoll_test.h",
+        "quiche/epoll_server/platform/api/epoll_thread.h",
+        "quiche/epoll_server/platform/api/epoll_time.h",
+    ],
+    visibility = ["//visibility:public"],
+    deps = ["@envoy//test/extensions/quic_listeners/quiche/platform:epoll_server_platform_impl_lib"],
+)
+
+cc_library(
+    name = "epoll_server_lib",
+    testonly = 1,
+    srcs = [
+        "quiche/epoll_server/fake_simple_epoll_server.cc",
+        "quiche/epoll_server/simple_epoll_server.cc",
+    ],
+    hdrs = [
+        "quiche/epoll_server/fake_simple_epoll_server.h",
+        "quiche/epoll_server/simple_epoll_server.h",
+    ],
+    copts = envoy_copts("@envoy") + ["-Wno-error=unused-parameter"],
```

Comment:
Instead of ignoring it, can we fix the unused-parameter warning in epoll_server code? We don't need to change envoy_build_system.bzl that way.

Reply:
SGTM, please move "-Wno-error=unused-parameter" into a global list and use it in here and epoll_server_test.

Reply:
In case you missed it, this comment still needs to be resolved. @danzh2010

Reply:
Where to put the global list? envoy_build_system.bzl?

Reply:
In this file(bazel/external/quiche.BUILD), since it's only needed by quiche build rules.

Reply:
done, and change it to -Wno-unused-parameter to suppress the warning entirely. Otherwise there will be too many warning output.